Mathlib/GroupTheory/SpecificGroups/Dihedral.lean

```diff
@@ -134,6 +146,9 @@ theorem r_one_pow (k : ℕ) : (r 1 : DihedralGroup n) ^ k = r k := by
     norm_cast
     rw [Nat.one_add]
 
+theorem r_one_zpow (k : ℤ) : (r 1 : DihedralGroup n) ^ k = r k := by
```

Comment:
I think this should be simp as well.

Reply:
Thinking about this a bit more, I think it is better to provide `r i ^ k = r (k * i)` as a simp lemma.

Reply:
I think that can be left for future work, and in that future PR the `pow` and `zpow` lemmas can be adjusted at the same time.